### DISCUSSION TITLE
Include a new database in cancer scout?

### DISCUSSION THREAD

**Discussion by heronikdin** [Clinical-Genomics/scout#5287] [Ideas]
Hi! 

As you know we have many databases in cancer scout which is fantastic! We have found a new onw directly connected to drugs/resistance etc, it its really great and I wonder if it can be included? 
 here is the page: https://www.cancergenomeinterpreter.org/biomarkers 

This variant https://scout.scilifelab.se/cust087/F0038753/8035e674a40bea2d3563044e30951c6f in a recent case is found in this database but not in other cancer databases that we have on scout. It is a really "nasty" mutation with resistance to the main drug being used for a particular leukemia. 

If it is possible to put the database in, maybe we can have the link visible here: 
![image](https://user-images.githubusercontent.com/62098218/110907957-6c054e80-830e-11eb-95a8-c69e8048978a.png)

Hälsningar Hero!

**Comment by dnil** [comment]
Well, if you are sure about it we can try. We skipped it because it didn't seem like it has been updated the last few years, and to be honest their interface is difficult at best unless we get the variants annotated with their collection from a pipeline. Could you help us by posting a link to the page in cgi you would like to land on when you click the link in Scout?

**Comment by heronikdin** [comment]
Hej Daniel! 

So we used https://www.mtbp.org/UploadSequencingDataPublic.php (the portal we will discuss with David togheter), and by writing the varaint and gene: 

![image](https://user-images.githubusercontent.com/62098218/110911261-c0123200-8312-11eb-9c73-4de12711917d.png)

This takes us here: 

![image](https://user-images.githubusercontent.com/62098218/110911357-e20bb480-8312-11eb-9ff7-ac299ac807ec.png)

with a direct link to the exact variant in CGI. 

I dont know if we should work to implement CGI into scout or actually wait for the meeting with David, now that I think of it?

**Comment by heronikdin** [comment]
Ideally we would like to come to the exact variant link, but I dont know if that is feasable?

**Comment by dnil** [comment]
The easiest and most consistent would be to have this annotated together with everything else in Balsamic and just shown in Scout.

The resource is downloadable as a very small `.csv` file, with just a few thousand variants. Worst case we could just import these to Scout and show.  Coyote used to do it this way. Or you could import them as "Managed variants" if you wish.

There is also a somewhat cumbersome API on the service, very similar to the interface on MTB, where we *could* query for variants, but again, I don't think that is the best way to go.

More concerning, this resource has actually not updated for the last three years. To me that says it is no longer active, but we can likely be enlightened soon. A user named DTamborero seems to have curated a few of the variants around 2016.

Btw, is the whole "primary" situation for ABL1 well understood? At first glance it seems HGNC flag a different transcript primary than you and most of these legacy cancer resources? If the new transcript is the future, and the rest is well understood legacy, that is fine, but if HGNC curators are in error, I'm sure they would appreciate input! They seem nice, and are not that many.. 😊

**Comment by dnil** [comment]
Let me also plug the VarSome db - clicking that V takes you to:
![Screenshot 2021-03-12 at 14 24 04](https://user-images.githubusercontent.com/758570/110950218-dc7a9280-8343-11eb-935e-be801901a0cb.png)
(I understand they are different issues, just since you mentioned the other links, and not this.. 😊)

**Comment by heronikdin** [comment]
Hi! 

Regarding your first answer, I think I followed but not all of it.. eventhough the site has not been updated for a longtime, the ABL1 variants are very much the same to this date (we know this by looking at recent publications). And we found this site through David Tamoboreros MTB portal.

We have seen the varsome link of course, but what we generally are missing is the actionable targets and this page https://www.cancergenomeinterpreter.org/biomarkers was kind of nice. If it is too outdated, maybe we can skip this actually and we can check the site manually? But it had many hotspot genes and variants which are good to know what the actionability is.

**Comment by dnil** [comment]
Two things here, and one does not invalidate the other.

I hear good info on that site, even if old. It is sad that it is not incorporated into like onkokb - which does sort of list it, but only show "no treatment", which is probably correct but less informative than "resistant". What do you say we chat with DT and consider options after understanding the status of this db better?

Then the variant in question. My reading is that CGI has chr9:g.133748270T>C, and you have 9_133748272_C_A. Much like VarSome I would argue that it is a Strong reason, but no absolute, for arguing pathogenicity. You understand this as well as I, but for the completeness of argument, assume for instance the real cause of disease for chr9:g.133748270T>A was actually a splice change, and the aa missensse is just a conceptual byproduct. It is nice to find aa-change similar variants, but it is a different feature than finding the same variant.
